README.md

```diff
@@ -72,19 +72,19 @@ and [Open3D C++ interface](http://www.open3d.org/docs/release/cpp_api.html).
 
 ## Open3D-Viewer app
 
-<img align="left" width="480" src="https://raw.githubusercontent.com/intel-isl/Open3D/master/docs/_static/open3d_viewer.png">
+<img width="480" src="https://raw.githubusercontent.com/intel-isl/Open3D/master/docs/_static/open3d_viewer.png"/>
 
 Open3D-Viewer is a standalone 3D viewer app available on Ubuntu and macOS.
 Please stay tuned for Windows.
 
 Download Open3D Viewer from the
```

Comment:
I would merge this with the previous paragraph. Now if we are not doing the left-right alignment, one sentence per paragraph seems a bit sparse.